tests/sync/session/session_util.cpp

```diff
@@ -0,0 +1,31 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2017 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#include "session_util.hpp"
+
+namespace realm {
+    bool sessions_are_active(const SyncSession& session)
+    {
+        return session.state() == SyncSession::PublicState::Active;
```

Comment:
Not sure there's any reason to put these in a .cpp file when the header pulls in all of the other headers this needs anyway.